worker/send_sms.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function(notifier_messager) {
+  var LUMBERYARD_EVENT = "send_sms";
+
+  return function(id, event, cb) {
+    if (event.event_type !== "send_sms") {
```

Comment:
Are we localizing the text on the webmaker.org side of things? I kinda feel like we should have an event for each kind of sms we want to send, but maybe I'm just being overly cautious.